mcs/class/corlib/System/TimeZoneInfo.cs

```diff
@@ -119,6 +119,8 @@ static TimeZoneInfo CreateLocal ()
 			} catch {
 				try {
 					return FindSystemTimeZoneByFileName ("Local", Path.Combine (TimeZoneDirectory, "localtime"));	
+				} catch (TimeZoneNotFoundException) {
+					return Utc;
 				} catch {
 					return null;
 				}
```

Comment:
We have to.

A UNIX machine without $TZ and /etc/localtime runs on UTC.

Maybe we should get rid of the second `return null;` because there should be only `InvalidTimeZoneException` thrown, and it should (in my opinion) fall through.